client/sites/overview/components/migration-overview/components/migration-started-difm.tsx

```diff
@@ -24,6 +16,16 @@ export const MigrationStartedDIFM = ( { site }: { site?: SiteDetails } ) => {
 			<div className="migration-started-difm">
 				<h2 className="migration-started-difm__title">{ translate( 'What to expect' ) }</h2>
 				<ul className="migration-started-difm__list">
+					<li className="migration-started-difm__item">
+						<div className="migration-started-difm__icon-wrapper">
+							<Icon icon={ envelope } className="migration-started-difm__icon" size={ 30 } />
+						</div>
+						<span>
+							{ translate(
+								"You'll receive an email from our team with more details on the process and letting you know when the migration is underway."
+							) }
+						</span>
+					</li>
```

Comment:
Minor note: in future, it may be worth pushing a lot of this content into a local (sub)component to keep the structure easier to follow and ensure that we don't have minor discrepancies across items. 
**NOTE:** We should not tackle this as part of this PR.

e.g. if we called this `MigrationStartedItem`, we'd be able to express this new entry as follows:
```suggestion
					<MigrationStartedItem
						icon={ envelope }
						text={ translate(
							"You'll receive an email from our team with more details on the process and letting you know when the migration is underway."
						) }
					/>
```

Reply:
That's a good thought. I can tackle that in a follow-up.